WRITING_DOCS.md

```diff
@@ -9,8 +9,6 @@
   - [Divider](#divider)
   - [Grid](#grid)
```

Comment:
I can't comment on the getting start page, so I'm just putting this comment here.

The layout changed from bentos to a list on the "Getting Started" page.
![image](https://github.com/icerpc/icerpc-docs/assets/25963603/b6b9535c-52bc-472b-a979-79c7521cdab0)
I found the bentos better, since then you don't need to scroll down the page. And the text fits comfortably.
With the new list, alot of space feels wasted.

Reply:
Hmm strange bug, will see if I can reproduce. This PR is not intending to change any of the rendering so anything like this is a bug.

Reply:
Can you confirm if this is still happening? I cannot reproduce.

![Screenshot 2023-09-25 at 3 38 10 PM](https://github.com/icerpc/icerpc-docs/assets/14209515/c8da48a0-369e-431c-9c2b-919e2d4e1ce2)

Reply:
I actually think the new behavior is intended. We don't really have a tablet mode really (the search box has its own css that does whatever it wants for the most part).
So the app can really be thought of as [1] mobile [2] vertical tablet (same as mobile) [3] tablet (no right nav) [4] desktop. 

This is how we use tailwind with its default settings `"example-class md:example-class-tablet lg:example-class-full"`. Given this it is consistent with the rest of the UI in the application even though really we could make it take more space on vertical tablet [2].

Reply:
Ill make the change for now though as its more consistent with main.